### PR TITLE
feat: Uses out-of-band protocol instead of didexchange

### PR DESCRIPTION
**Pull request by soluchok**
Uses out-of-band protocol instead of didexchange.

Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

**Comment by llorllale** [comment]
@soluchok have you tested this in the sandbox with updated adapter and edge-agent images?

**Comment by soluchok** [comment]
> @soluchok have you tested this in the sandbox with updated adapter and edge-agent images?

I tested only edge-agent. Also, I asked @sudeshrshetty to help me to check it. sandbox I did not test.

**Comment by llorllale** [comment]
@soluchok 

> > @soluchok have you tested this in the sandbox with updated adapter and edge-agent images?
> 
> @llorllale I tested only edge-agent. Also, I asked @sudeshrshetty to help me to check it. sandbox I did not test.

Please test in the sandbox and make sure the DIDComm flow works.

File: cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js [hunk 1]
```diff
@@ -1,120 +0,0 @@
...
-import axios from 'axios';
-
-const routerCreateInvitationPath = `/connections/create-invitation`
-
-/**
- * AgentMediator provides aries mediator features
- * @param aries agent instance
- * @class
- */
-export class AgentMediator {
```
Comment (sudeshrshetty) [inline cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js:16]: Why mediator is deleted?

Comment (soluchok) [reply cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js:16]: We don't need it. The mediator is accessible through `vuex` actions.
https://github.com/trustbloc/edge-agent/blob/master/cmd/user-agent/src/store/modules/mediator.js#L39

File: cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js [hunk 2]
```diff
@@ -20,20 +23,24 @@ export class AgentMediator {
...
         let invitation = await createInvitationFromRouter(endpoint)
-        let aries = this.aries
-        let conn = await aries.didexchange.receiveInvitation(invitation)
 
-        await this.waitFor(conn.connection_id, 'invited', function () {
-            return aries.didexchange.acceptInvitation({
-                id: conn.connection_id
-            })
+        let conn = await this.aries.outofband.acceptInvitation({
+            my_label: 'agent-default-label',
```
Comment (sudeshrshetty) [inline cmd/user-agent/src/pages/chapi/wallet/didcomm/connections.js:28]: may be a follow up on label, created https://github.com/trustbloc/edge-agent/issues/276

**Pull request merged**